bgpd/bgpd.c

```diff
@@ -4574,7 +4574,7 @@ static const struct peer_flag_action peer_flag_action_list[] = {
 	{PEER_FLAG_AIGP, 0, peer_change_none},
 	{PEER_FLAG_GRACEFUL_SHUTDOWN, 0, peer_change_none},
 	{PEER_FLAG_CAPABILITY_SOFT_VERSION, 0, peer_change_none},
-	{PEER_FLAG_CAPABILITY_FQDN, 0, peer_change_reset},
+	{PEER_FLAG_CAPABILITY_FQDN, 0, peer_change_none},
 	{0, 0, 0}};
```

Comment:
are you sure ? cisco does resets upon config ?

Reply:
why are you changing the behaviour ?
I think an other product vendor like cisco does perform a reset when the fqdn param is modified.
so we want to keep this behaviour identical.
what is the reason why you set it to none?

Reply:
Because dynamic capabilities logic makes no sense then. If you have dynamic capabilities, the reset is not necessary.